.circleci/config.yml

```diff
@@ -245,7 +245,7 @@ jobs:
             # multiple tests in one file)
             # -rA prints summary in the end
             # -n4 uses four processes to run tests via pytest-xdist
-            pipenv run pytest --junitxml=$TEST_OUTPUT/junit.xml --tb=short -s --verbose -rA $TEST_SELECTION $EXTRA_PARAMS
+            pipenv run pytest --junitxml=$TEST_OUTPUT/junit.xml --tb=short --verbose -rA $TEST_SELECTION $EXTRA_PARAMS
```

Comment:
It disables instant test output display, as test outputs are mixed with each other when running in parallel. Logs in python tests are captured and always displayed after all tests are completed.

Reply:
Yep, added some info why we don't need to use `-s` in CI